vllm/worker/model_runner.py

```diff
@@ -1497,9 +1498,12 @@ def capture_model(self, kv_caches: List[List[torch.Tensor]]) -> None:
                         graph_runner)
 
         end_time = time.perf_counter()
+        end_free_gpu_memory = torch.cuda.mem_get_info()[0]
         elapsed_time = end_time - start_time
+        cuda_graph_size = start_free_gpu_memory - end_free_gpu_memory
         # This usually takes < 10 seconds.
         logger.info("Graph capturing finished in %.0f secs.", elapsed_time)
+        logger.info("CUDA graph takes %.2f GiB", cuda_graph_size / GiB_bytes)
```

Comment:
Could we please combine these logs? Such as 
```python
logger.info("Graph capturing finished in %.0f secs, took %.2f GiB", elapsed_time, cuda_graph_size / GiB_bytes)
```

Reply:
thanks for the comment! Merged and reformatted.